cmd/localCI/Gopkg.lock

```diff
@@ -1,6 +1,13 @@
-memo = "6741108163c25829de2a1320721785eac9e6a0af6714a59b8e8ea522382ed194"
+memo = "5217388a073fbb9d27a4d6da44598eaa256a9468e261fe45610cf66f6f338fe0"
 
 [[projects]]
+  name = "github.com/01org/ciao"
```

Comment:
Just to ask, do you need the top level here, or could/should it just pull the ciao-down path https://github.com/01org/ciao/tree/master/testutil/ciao-down ?